MetaMask/metamask-extension
# Add CachedBalancesController state migration

## **Description**
https://github.com/MetaMask/metamask-extension/pull/22276 removed the `CachedBalancesController` and should have included this migration to migrate the state to AccountsTrackerController and delete the old controller state.

## **Manual testing steps**

1. Build this branch where you'd had a wallet previously built on develop (with several accounts across different chains)
2. Move between accounts and chains. 
3. You should see your balances without a loading spinner

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
